packages/components/src/internal/sampleModels.ts

```diff
@@ -1,14 +1,45 @@
 import { List, Map, OrderedMap } from 'immutable';
 import { ComponentType } from 'react';
+import { Query } from '@labkey/api';
 
 import { QueryModel } from '../public/QueryModel/QueryModel';
-import { RequiresModelAndActions } from '../public/QueryModel/withQueryModels';
+import { InjectedQueryModels, RequiresModelAndActions } from '../public/QueryModel/withQueryModels';
 import { SchemaQuery } from '../public/SchemaQuery';
 
+import { TabbedGridPanelProps } from '../public/QueryModel/TabbedGridPanel';
+
 import { ComponentsAPIWrapper } from './APIWrapper';
 import { User } from './components/base/models/User';
 import { EntityDataType } from './components/entities/models';
 import { SampleGridButtonProps } from './components/samples/models';
+import { ALIQUOT_FILTER_MODE } from './components/samples/constants';
+
+export interface SamplesTabbedGridPanelComponentProps {
```

Comment:
If this interface is left duplicated please put a comment at the top of this one and the one declared in `@labkey/premium` pointing out how they both need to be updated. Left a comment on the `@labkey/premium` PR regarding whether we should be duplicating these.

Reply:
I exported this from the package and used it in the @labkey/premium case. Thanks